Promote Pod Disruption Budgets to GA (#27053)
kubernetes/website: content/en/docs/tasks/run-application/configure-pdb.md

<!-- overview -->

> {{< feature-state for_k8s_version="v1.5" state="stable" >}}

Reviewer: ```suggestion
{{< feature-state for_k8s_version="v1.21" state="stable" >}}
```

## {{% heading "prerequisites" %}}

> {{< version-check >}}

Reviewer: I recommend (not essential, but a good idea) that we ask people using older Kubernetes versions to check the documentation for that Kubernetes version.

Author: Not sure I understand what we are looking for here. Is there an example I can look at?

of the number of pods from that set that can be unavailable after the eviction. 
It can be either an absolute number or a percentage.

{{< note >}}

> The behavior for an empty selector differs between the policy/v1beta1 and policy/v1 apis for
> Pod Disruption Budgets. For policy/v1beta1 an empty selector will match zero pods, while
> for policy/v1 it will select all pods in the namespace.

Reviewer: style nit:
```suggestion
The behavior for an empty selector differs between the policy/v1beta1 and policy/v1 APIs for
PodDisruptionBudgets. For policy/v1beta1 an empty selector matches zero pods, while
for policy/v1 an empty selector matches every pod in the namespace.
```

of the number of pods from that set that can be unavailable after the eviction. 
It can be either an absolute number or a percentage.

{{< note >}}
The behavior for an empty selector differs between the policy/v1beta1 and policy/v1 APIs for
PodDisruptionBudgets. For policy/v1beta1 an empty selector matches zero pods, while
for policy/v1 an empty selector matches every pod in the namespace.
{{< /note >}}

> {{< note >}}

Reviewer: Consider omitting the “For versions 1.8 and earlier” note as v1.8 is long out of support.

Author: Removed the section.
